assets/app/views/createRoute.html

```diff
@@ -0,0 +1,52 @@
+<div class="content">
+  <div class="container">
+    <div class="col-md-12">
+      <ol class="breadcrumb">
+        <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
+        <li class="active"><strong>Add Route</strong></li>
+      </ol>
+      <alerts alerts="alerts"></alerts>
+      <h1>Add Route</h1>
+      <div ng-if="serviceName">
+        <span class="help-block">Create a route for service {{serviceName}}.</span>
+      </div>
+      <div ng-if="!serviceName">
+        <span class="help-block">Select a service and enter route details.</span>
+      </div>
+      <div ng-show="!serviceName && !services">Loading...</div>
+      <div ng-show="serviceName || services">
+        <form name="createRouteForm">
+          <fieldset ng-disabled="disableInputs">
+            <div ng-if="!serviceName" class="form-group">
+              <label for="service-select" class="required">Service</label>
+              <select
+                  id="service-select"
+                  ng-model="routing.service"
+                  ng-options="service as service.metadata.name for service in services track by (service | uid)"
+                  required
+                  class="form-control"
+                  aria-describedby="service-help">
+              </select>
+              <div>
+                <span id="service-help" class="help-block">Service to route to.</span>
+              </div>
+              <div ng-if="!services.length" class="has-error">
+                <span class="help-block">No services for route.</span>
```

Comment:
probably need a better message here, "There are no services in your project to create a route to." or something like that